nixodmlconverter/convert.py

```diff
@@ -466,10 +475,17 @@ def convert(filename, mode='append'):
 def main(args=None):
     parser = docopt(__doc__, argv=args, version=VERSION)
 
-    files = parser['FILE']
-    print(files)
-    for curr_file in files:
-        convert(curr_file)
+    file_or_dir = parser['FILE_OR_DIR']
+    mode = parser['MODE'] if parser['MODE'] else 'append'
+    if not os.path.splitext(file_or_dir)[1]:
+        file_type = parser['TYPE'] if parser['TYPE'] else "nix"
```

Comment:
Check if a valid type has been provided, otherwise switch to default and print a corresponding message.